docs/rackhd/event_notification.rst

```diff
@@ -1,105 +1,264 @@
+                    "pid": 26633,
+                    "lastUpdate": "2016-07-13T14:23:35.784Z",
+                    "currentTime": "2016-07-13T14:23:45.627Z",
+                    "nextUpdate": "2016-07-13T14:23:55.627Z",
+                    "cpuUsage": {
+                        "system": 92810,
+                        "user": 1545498
+                    },
+                    "memoryUsage": {
+                        "heapTotal": 71938048,
+                        "heapUsed": 45444240,
+                        "rss": 98926592
+                    },
+                    "platform": "linux",
+                    "release": {
+                        "headersUrl": "https://nodejs.org/download/release/v6.3.0/node-v6.3.0-headers.tar.gz",
+                        "name": "node",
+                        "sourceUrl": "https://nodejs.org/download/release/v6.3.0/node-v6.3.0.tar.gz"
+                    },
+                    "title": "node",
+                    "uid": 0,
+                    "versions": {
+                        "ares": "1.10.1-DEV",
+                        "http_parser": "2.7.0",
+                        "icu": "57.1",
+                        "modules": "48",
+                        "node": "6.3.0",
+                        "openssl": "1.0.2h",
+                        "uv": "1.9.1",
+                        "v8": "5.0.71.52",
+                        "zlib": "1.2.8"
+                    }
+                }
+            },
+            "properties": {
+                "content_type": "application/json",
+                "type": "Result"
+            }
+        }
     }
 
-The payload attributes of node event:
+Events via AMQP
+~~~~~~~~~~~~~~~~~~~~
 
-========= ====== =================================
-Attribute Type   Description
-========= ====== =================================
-type      String `type` is `'node'` for node events
-action    String The `action` that was performed by node event, See actions_ for more details.
-nodeId    String The node's unique identifier
-nodeType  String The node type could be `compute`, `pdu`, `switch`, `mgmt`, `enclosure`, it's related to `action`
-uri       String it's **optional**
-========= ====== =================================
+AMQP Exchange and Routing Key
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The change of resources managed by RackHD could be retrieved from AMQP messages.
+
+- Exchange: **on.events**
+- Routing Key **<type>.<action>.<severity>.<typeId>.<nodeId>**
+
+ALl the fields in routing key exists in the common event payloads common_event_payload_format_.
+
+Examples of routing key:
+
+Heartbeat event routing key of on-tftp service:
+
+.. code-block:: REST
+
+    heartbeat.updated.information.kickseed.example.com.on-tftp
+
+Polleralert sel event routing key:
+
+.. code-block:: REST
+
+    polleralert.sel.updated.critical.44b15c51450be454180fabc.57b15c51450be454180fa460
+
+Node discovered event routing key:
+
+.. code-block:: REST
 
-.. _actions:
```

Comment:
Why are we removing a list of the possible events that can be received? Where else is this documented other than scanning the messenger code?